goldmane/pkg/aggregator/index_test.go

```diff
@@ -21,9 +21,34 @@ import (
 
 	"github.com/projectcalico/calico/goldmane/pkg/internal/types"
 	"github.com/projectcalico/calico/goldmane/pkg/internal/utils"
+	"github.com/projectcalico/calico/goldmane/proto"
 	"github.com/projectcalico/calico/libcalico-go/lib/logutils"
+	"github.com/projectcalico/calico/libcalico-go/lib/set"
 )
 
+type simpleLogAggregatorStub struct {
+	diachronics []*types.DiachronicFlow
```

Comment:
not really a problem, but seems like this structure would be a lot easier to implement if it just had a `set.Set` here already - one function converts to a Set anyway, and the other does a lookup by iterating the items instead of just using the Set to access the value.

Reply:
Ya probably, I'll think about adding this now.

Reply:
I think I'm just going to leave this for now to get this through.